src/plan/diff_action.go

```diff
@@ -14,6 +14,7 @@ const (
 	ActionSaveRemote
 	ActionDeleteLocal
 	ActionDeleteRemote
+	ActionMarkDeletedRemote
```

Comment:
Nova akcia, ktora prida pred nazov `[TO DELETE]` tj vymaze objekt z dev branche.